gdnative-core/src/nativescript/init/property/hint.rs

```diff
@@ -116,30 +119,31 @@ where
 /// ```
 #[derive(Clone, Eq, PartialEq, Debug, Default)]
 pub struct EnumHint {
-    values: Vec<String>,
+    values: HashMap<String, i32>,
```

Comment:
My edit on the post came a bit late, maybe you missed it:
> Unless we need properties of a map somewhere, I would keep things simple and stay with `Vec`.

It's probably better for such little-used things, as `Vec` is much cheaper (performance- and memory-wise).